pkgs/by-name/te/temp-throttle/package.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, stdenv, fetchFromGitHub }:
```

Comment:
Please format your sources using `nixfmt-rfc-style`.

Reply:
I have converted all of the code to nixfmt, except at top of the service module section, with creation of the temp-throttle.conf file. I am inclined to believe my code at that section is more easily read by a human than the output generated by nixfmt. If you disagree, then I shall resubmit with all code generated by nixfmt.

Reply:
All sources have been reformatted. 🙂